ocs_ci/helpers/managed_services.py

```diff
@@ -694,3 +694,71 @@ def verify_faas_provider_storagecluster_storages(sc_data):
 
     log.info(f"The values in the storage profile {default_sp_name} are correct")
     log.info("Finish Verifying all the storages in the provider faas storagecluster")
+
+
+def create_toolbox_on_faas_consumer():
```

Comment:
In functions when we switch from one cluster to another, we had better use the decorator `@switch_to_orig_index_at_last` from the file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/decorators.py.

Reply:
Added the decorator.